app/controllers/callbacks_controller.rb

```diff
@@ -192,7 +208,7 @@ def validate_discord_callback
 
     head :unauthorized unless is_valid.all?
```

Comment:
what is is_valid and could it be moved somewhere else?  it doesn't sound like it matters for defining @user_connections and if it does... does it matter that if @user_connections is already defined we won't check it?